xml/vt_qemu_ga.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-qemu-ga">
+ <title>&qemu; Guest Agent</title>
+ <para>
+  The &qemu; guest agent (GA) runs inside the &vmguest; and allows the &vmhost;
+  to run commands int the guest operating system via &libvirt;. It supports
+  many functions&mdash;for example, getting details about guest file systems,
+  freezing and thawing file systems, or suspending or rebooting a guest.
+ </para>
+ <para>
+  &qemu; GA is included in the <package>qemu-guest-agent</package> package and
+  is installed, configured, and activated by default in &productname;
+  &productnumber;.
+ </para>
+ <sect1 xml:id="cha-qemu-ga-libvirt-general">
+  <title>Running &qemu; GA Commands</title>
+
+  <para>
+   &qemu; GA includes many native commands that do not have their &libvirt;
+   counterparts. Refer to <xref linkend="cha-qemu-ga-moreinfo"/> for the source
+   of their complete list. You can run all of the &qemu; GA commands by using
```

Comment:
```suggestion
   counterparts. Refer to <xref linkend="cha-qemu-ga-moreinfo"/> for the 
   complete list. You can run all of the &qemu; GA commands by using
```

Reply:
I am not sure of this, but "source of their complete list" sounds strange to me.

If the link-end does not contain the complete list, but merely (for example) a link to it, then perhaps:

... refer to $linkend to find the complete list.